script/install-nginx.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright 2018-2022 the Kubeapps contributors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "Instaling Nginx"
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+helm repo update
+helm install -n nginx-ingress --create-namespace nginx-ingress ingress-nginx/ingress-nginx
```

Comment:
I guess we don't need the annotations we use to add for Kubeapps itself, no?

Reply:
Do you mean the buffers setup? Not sure what values come installing Nginx with Helm.
If annotations don't cause harm, I would leave them by now.